uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerAuthenticatorUpdatedVersionImpl.java

```diff
@@ -138,6 +138,19 @@ public void performSharedDeviceRegistration(@NonNull final String username,
 
         isInSharedDeviceMode = true;
     }
+
+    @Override
+    public void performSharedDeviceRegistrationDontValidate(@NonNull final String username,
+                                                @NonNull final String password) {
+        Logger.i(TAG, "Performing Shared Device Registration for the given account..");
```

Comment:
Let's change this log message to indicate the "Don't Validate" aspect.

Reply:
Why do we need a performSharedDevieRegistration method where we don't check. Is this related to Samsung device automation? Are we not seeing the screen at all or are we failing to find the correct UI element because we don't have the resource id.

Reply:
If you are having trouble using UiAutomatorViewer on your samsung device still, let's document this in your rampup notes. This may be useful for someone else later down the line.

Reply:
> Why do we need a performSharedDevieRegistration method where we don't check. Is this related to Samsung device automation? Are we not seeing the screen at all or are we failing to find the correct UI element because we don't have the resource id.

The original performSharedDeviceRegistration method is designed to pass by validating the device is in shared mode, but the test case I'm implementing is supposed to fail when performing shared device with a non-admin account. Thus the need for a new performSharedDevice without validation.

Reply:
I see, good to know. Thanks for clarifying that